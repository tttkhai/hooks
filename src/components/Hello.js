import React from "react";

function Hello() {
  return <div>showing Hello</div>;
}

export default Hello;
