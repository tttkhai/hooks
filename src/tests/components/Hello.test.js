import React from 'react'
import { shallow } from "enzyme";
import Hello from "../../components/Hello";
import toJSON from 'enzyme-to-json';

test("Hello should be rendered correctly", () => {
  const wrapper = shallow(<Hello />);
  // expect(wrapper).toMatchSnapshot();
  expect(toJSON(wrapper)).toMatchSnapshot()
});

test("should return 1 for length", () => {
  const wrapper = shallow(<Hello />);
  expect(wrapper.find('div').text()).toBe('showing Hello');
});

