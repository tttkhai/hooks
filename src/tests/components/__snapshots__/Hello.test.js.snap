// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello should be rendered correctly 1`] = `
<div>
  showing Hello
</div>
`;
